FlagsmithClient/src/test/java/com/flagsmith/FeatureFlagTests.kt

```diff
@@ -1,15 +1,41 @@
 package com.flagsmith
 
-import junit.framework.Assert.*
+import com.flagsmith.mockResponses.MockEndpoint
+import com.flagsmith.mockResponses.mockResponseFor
 import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
 import org.junit.Test
+import org.mockserver.integration.ClientAndServer
 
 class FeatureFlagTests {
 
-    private val flagsmith = Flagsmith(environmentKey = System.getenv("ENVIRONMENT_KEY") ?: "", enableAnalytics = false)
+    private lateinit var mockServer: ClientAndServer
+    private lateinit var flagsmith: Flagsmith
+
+    @Before
+    fun setup() {
+        mockServer = ClientAndServer.startClientAndServer()
+        flagsmith = Flagsmith(
+            environmentKey = System.getenv("ENVIRONMENT_KEY") ?: "",
```

Comment:
Is the `System.getenv("ENVIRONMENT_KEY")` left here so the tests can still be run as integration tests if we want to in the future?

Reply:
Ah, no. I just forgot to remove it. I'll do that now.